libs/components/config/src/lib/config-remote-modules.ts

```diff
@@ -0,0 +1,11 @@
+import { SkyuxConfigRemoteEntryReference } from './config-remote-entry-reference';
+
+/**
+ * Information about remote modules referenced by this application.
+ */
+export interface SkyuxConfigRemoteModules {
+  /**
+   * A collection of remote modules referenced by this application.
+   */
+  referenced?: SkyuxConfigRemoteEntryReference[];
```

Comment:
With this setup, a consumer might get themselves into this situation:
```
"remoteModules": {
  "referenced": [
    {
      "app": "foo",
      "version: 1,
      "exposedModules": ["./SomeComponent"]
    },
    {
      "app": "foo",
      "version: 2,
      "exposedModules": ["./SomeOtherComponent"]
    }
  ]
}
```
Acceptable?

Reply:
That's pretty much the same issue :( How about this?

Reply:
That's pretty much the same issue :( I could make the app the object's key, but I feel like that's harder to discover for the author of the config file. I'll play around with that and may push those changes.